lib/components/viewers/route-details.js

```diff
@@ -1,18 +1,17 @@
 import React, { Component } from 'react'
-import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { FormattedMessage, injectIntl } from 'react-intl'
+import PropTypes from 'prop-types'
```

Comment:
Sort imports

Reply:
Sorted in 
092b53f aside from React which I think needs to come first